pom.xml

```diff
@@ -63,9 +67,30 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>net.devh</groupId>
+            <artifactId>grpc-spring-boot-starter</artifactId>
+            <version>${grpc-spring-boot-starter.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>${javax-annotation.version}</version>
+        </dependency>
+
     </dependencies>
 
     <build>
+
+        <extensions>
+            <extension>
+                <groupId>kr.motd.maven</groupId>
```

Comment:
Why do you need this for?

Reply:
This is recommended to automatically generate a classifier for the current OS and architecture. This is use for protobuf plugin to get OS classifier.